README.md

```diff
@@ -6,15 +10,15 @@
 
 <!-- [END badges] -->
 
-<img src="https://user-images.githubusercontent.com/10379601/29446482-04f7036a-841f-11e7-9872-91d1fc2ea683.png" height="200" align="right">
+<img src="https://user-images.githubusercontent.com/10379601/29446482-04f7036a-841f-11e7-9872-91d1fc2ea683.png" height="200" align="right"/>
 
-###### [API](https://github.com/puppeteer/puppeteer/blob/v15.2.0/docs/api.md) | [FAQ](#faq) | [Contributing](https://github.com/puppeteer/puppeteer/blob/main/CONTRIBUTING.md) | [Troubleshooting](https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md)
+###### [API](https://puppeteer.github.io/puppeteer/api) | [FAQ](https://puppeteer.github.io/puppeteer/faq) | [Contributing](https://puppeteer.github.io/puppeteer/contributing) | [Troubleshooting](https://puppeteer.github.io/puppeteer/troubleshooting)
```

Comment:
Why don't we link to the markdown documentation in docs/api?

Reply:
should not we link to pttr.dev? What's the status/plan there?

Reply:
Done.